src/Service/AssentService.php

```diff
@@ -150,24 +150,24 @@ public function updateAssentHandler(array $data, array $config)
             throw new NotFoundHttpException("The assent with id {$data['path']['id']} was not found.");
         }
 
-        $assentData = $assent->toArray();
+        if ($assent->getValue('status') === 'granted') {
+            // get brp person from the logged in user
+            $brpPersons = $this->cacheService->searchObjects(null, ['burgerservicenummer' => $this->security->getUser()->getPerson()], [$brpSchema->getId()->toString()])['results'];
+            $brpPerson  = null;
+            if (count($brpPersons) === 1) {
+                $brpPerson = $this->entityManager->find('App:ObjectEntity', $brpPersons[0]['_self']['id']);
```

Comment:
f.y.i.
je kan ook 
`$brpPersons = $this->cacheService->searchObjects(null, ['burgerservicenummer' => $this->security->getUser()->getPerson()], [$brpSchema->getId()->toString()]);
            $brpPerson  = null;
            if ($brpPersons['count'] === 1) {
                $brpPerson = $this->entityManager->find('App:ObjectEntity', $brpPersons['results'][0]['_self']['id']);`
doen